doc/data/ttn-lw-cli.yml

```diff
@@ -22,7 +22,7 @@ ttn-lw-cli:
     type: stringSlice
     shorthand: c
     usage: Location of the config files
-    default_value: '[.ttn-lw-cli.yml,$HOME/.ttn-lw-cli.yml,$HOME/Library/Application Support/.ttn-lw-cli.yml]'
+    default_value: '[.ttn-lw-cli.yml,/home/nejra/.ttn-lw-cli.yml,/home/nejra/.config/.ttn-lw-cli.yml]'
```

Comment:
This should stay `$HOME`. 

I specifically asked @benolayinka to make sure the template would avoid this (see https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/294/files#r604903025)